### ISSUE TITLE
accept photos that do NOT have names/badge numbers

### ISSUE THREAD

**Issue by redshiftzero** [lucyparsons/OpenOversight#723]
There are some law enforcement officers that are covering up their name and/or badge number when working protests and a lot of folks are taking photos of them. We should consider relaxing our photo submission to accept photos for any officer in uniform: even if their badge name or name is not exposed. 

In that scenario the modified workflow could be:

1. Not logged in users submits a photo as before.
2. Volunteers use cop classification to determine if the photo fits into one of three categories: (A) cop has name/badge number, (B) cop but no name or badge number, (C) no cop. I'm suggesting this as I do think we should allow volunteers to choose whether they are merely building the OpenOversight dataset by adding faces, OR they are trying to identify a potentially problem cop by matching him with a face that exists in our dataset. These seem like different cases.
3. For cops with name/badge number (case A), same workflow as we have now -> Cop identification for the purpose of building out our dataset. 
4. For cops _without_ name/badge number (case B) -> a very similar workflow (go through roster) but: we instruct users to match the face with one that exists in the dataset. For departments in states without facial recognition constraints, we can at some later point automate this step.

**Comment by ssempervirens** [comment]
Just a clarification - if people submit photos of officers without badges/names and we don't already have a photo/s of the officer, and so the face can't be matched to an existing photo, we just store the photos for the future, correct? 

Also, a big problem is that many officers in recent photos are wearing masks, so matching them to faces seems like it could be tricky.

**Comment by redshiftzero** [comment]
> if people submit photos of officers without badges/names and we don't already have a photo/s of the officer, and so the face can't be matched to an existing photo, we just store the photos for the future, correct?

Yep exactly. 

Hmm good point re: masks. We should research how reliable human face recognition is with faces covered by surgical masks - I _suspect_ there are enough facial characteristics to still produce good identifications, but more research needed.

**Comment by brianmwaters** [comment]
I like everything that's being suggested here, and I think we can do one better too. I think the unstated assumption we might be making is that we can't identify cops who don't have a name or badge number in the photo. This isn't the case in Vermont. We have extensive on-the-ground experience with the BPD, and can identify most of them by face. And we have identified a large number of cops in other departments based on copaganda uploaded to social media, local news reports, etc.

Seems like this sort of thing would be more difficult in large cities with big departments, but I think the knowledge is out there, and the right community members would be able to identify a lot of cops. There might be challenges with mis-identifications and traceability (how did we come to the conclusion that face A belongs to cop B), but I don't know if those are insurmountable.

**Comment by brianmwaters** [comment]
Or - I guess I should ask - is this something we've had problems with in the past?

**Comment by redshiftzero** [comment]
yeah good point - that's something that Berkeley copwatch folks told us too - they know all the cops by face. I'm thinking that it might make sense to have a lot of area coordinators for cities like Berkeley and BTV (this is something we can document for those cities) so y'all can just add photos/data as you see fit. We do track which user account associated picture X with officer profile Y so that should handle the traceability concern. We haven't had problems _yet_ but we implemented that just in case we needed to disable specific accounts and all their activities on the platform (i.e. in case of malicious users).
